content/blog/multi-lang-components/index.md

```diff
@@ -15,30 +16,38 @@ tags:
 ---
 
 Pulumi's infrastructure as code tooling combines the programming languages and tools you already know with the full power of cloud
-infrastructure. Until now, some Pulumi components for cloud infrastructure, like our popular [EKS package], were
-only available in a subset of the languages supported by Pulumi.
+infrastructure. Until now, some Pulumi components for cloud infrastructure, like our popular [EKS package] for Amazon's Elastic
+Kubernetes Service, were only available in a subset of the languages supported by Pulumi.
 
-Now, with multi-language components, anyone can author a Pulumi component in the language they prefer and make it available
-across all Pulumi languages. To celebrate, we've taken that [EKS package]–previously only available for TypeScript–
-and made it available in all four Pulumi languages: TypeScript, Python, .NET, and Go. Regardless of the language you choose, you can
-[manage EKS clusters] with Pulumi, starting with the [v0.22.0 release of the EKS package]. Check out our Modern Infrastructure
-Wednesday video to see it in action:
+Now, you can use the [EKS package]–previously only available for TypeScript–in all four Pulumi languages: TypeScript, Python, .NET,
+and Go. Regardless of the language you choose, you can [manage EKS clusters] with Pulumi, starting with the [v0.22.0 release of the
+EKS package]. Check out our Modern Infrastructure Wednesday video to see it in action:
 
 {{< youtube "gxLyAr0lUg0?rel=0" >}}
 
 <!--more-->
 
-EKS in all languages is just the start of the multi-language components story. Multi-language components enable all organizations to choose
-the right programming language, or languages, for each team and each product. Your infrastructure or platform teams can
-write components in Go and share them with a webapp team using TypeScript, or write in .NET and share with a machine learning team
-using Python or any other combination that suits your business. Similarly, any member of the worldwide Pulumi community can author
-a component in the language they're most comfortable with and share it with all Pulumi users.
+EKS in all languages is made possible by our ongoing work to enable "[multi-language components]". With multi-language components,
+any member of the worldwide Pulumi community will be able to author a Pulumi component in the language they're most comfortable
+with and share it with all Pulumi users: even those that use a different language than the component was authored in. Multi-language
```

Comment:
I like the idea of adding something here that talks about the amplification effects of writing in one language and duplicating into other languages. Something like

"Before multi-language components, unless you were willing to write the code in all of our languages, you would only be able to reach Pulumi users using the same language SDK. With multi-language components, writing in one language will now reach 3x as language platforms and countless new users, with no extra code needed!"